packages/fhir-converter/src/routes.js

```diff
@@ -225,9 +226,9 @@ module.exports = function (app) {
           delete resultMessage["invalidAccess"];
         }
         console.log(
-          `[patient ${patientId}] Took ${duration}ms / status ${result.status} to process file ${fileName}`
+          `[patient ${patientId}] Took ${duration}ms / status ${status} to process file ${fileName}`
         );
-        res.status(result.status);
+        res.status(status ?? 400);
```

Comment:
No need for the fallback here, right?

Reply:
oh woops. Missed it after i factored out the new const from this line 😬 good catch